chcemvediet/apps/inforequests/tests/test_models/test_deadline.py

```diff
@@ -0,0 +1,157 @@
+from django.test import TestCase
+
+from poleno.utils.date import naive_date, local_today
+
+from .. import InforequestsTestCaseMixin
+from ...models.deadline import Deadline
+
+
+class DeadlineTest(InforequestsTestCaseMixin, TestCase):
+    u"""
+    Tests ``Deadline`` model.
+    """
+    def _create_deadline(self, **kwargs):
+        _today = kwargs.pop(u'_today', local_today())
+        deadline = self._call_with_defaults(Deadline, kwargs, {
+            u'type': Deadline.TYPES.OBLIGEE_DEADLINE,
+            u'base_date': local_today(),
+            u'value': 8,
+            u'unit': Deadline.UNITS.WORKDAYS,
+            u'snooze': None,
+        })
+        deadline._today = _today
```

Comment:
Naco je toto dobre? Nie je praktickejsie pred volanim `_create_deadline` proste jumpnut v case, nech `local_today()` je to co chceme? Radsej globalne overridnut aktualny cas, nez len na jednom mieste zmenit hodnotu. Taketo ad-hoc overridy su nachylne na chyby.

Reply:
:ok: pouzil som timewarp. Dakujem za vysvetlenie. Osobne sa mi to zdalo prehladnejsie ked som zadaval cas pomocou `_today` attributu priamo pri vytvarani deadlinu.

Reply:
@viliambalaz Suhlasim, ze to vyzera prehladnejsie. Ale predstav si, ze ten kod chytis o par rokov. Budes si este pamatat, ake ad-hoc overridy si spravil? Ked na "cestovanie casom" mame standardny mechanizmus, tak je lepsie ho pouzivat systematicky vsade.